README.md

```diff
@@ -307,6 +312,10 @@ directory. You should add `it.only` to the test case you are working on to speed
 
 ## Release Notes
 
+#### 3.4.3
```

Comment:
This will be 3.5.0 since we are introducing new feature. Patch version increments should contain only bugfixes

Reply:
fixed